tests/e2e/init_test.go

```diff
@@ -12,9 +12,11 @@ import (
 
 var ctx context.Context
 var token string
+var cluster_id string
```

Comment:
```suggestion
var clusterID string
```

Just so that it will be aligned with the name conventions